src/app/routes/applications/governance-list/governance-list.component.ts

```diff
@@ -368,7 +370,27 @@ export class GovernanceListComponent implements OnInit, OnDestroy {
     this.dataSource.data = JSON.parse(JSON.stringify(this.origDatasource));
   }
 
+  createSubOrg(parentOrg: any) {
+    const dialogRef = this.dialog.open(NewOrganizationComponent, {
+      width: '600px',
+      data: {
+        viewType: ViewType.NEW,
+        parentOrg: JSON.parse(JSON.stringify(parentOrg)),
+        owner: parentOrg.owner
+      },
+      maxWidth: '100%',
+      disableClose: true
+    });
+
+    dialogRef.afterClosed()
+      .pipe(
+        filter(Boolean),
+        takeUntil(this.subscription$))
+      .subscribe(async () => await this.newSubOrg(parentOrg));
+  }
+
   async newSubOrg(parentOrg: any) {
```

Comment:
If this is only being called in component, can it be made private?
```suggestion
  private async newSubOrg(parentOrg: any) {
```

Reply:
This is used in template as well.
I refactored this once and that's why this regression happened. I didn't expected that parent component is using child component methods.
I created there components for actions, and those components are handling displaying dialogs etc. method like `newSubOrg` is used in template just to refresh the list after closing a dialog.

Reply:
Ok, fair enough 👍